### PR TITLE
Added myself to the 2023 folder

### PR DESCRIPTION
**Pull request by SAMAD101**

### CODE REVIEW

File: 2023/Abdul Samad Ansari (Sam)/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+## **Introduction**
+- Hello there, I'm Abdul Samad Ansari, you can call me Sam for short. I've been programming with Python language for the 4+ years now. I was first introduced to programming back in 8th grade and I'm in love ever since. I first learned about making basic webpages using HTML and CSS and, then shifted to Python to explore more. I started getting more involved in Open Source after coming to college. Last month (February), I learned about Django ORM, Redis and setting up a wagtail project or integrate wagtail to a django project. Currently, I'm looking into django and wagtail in order to learn more about the codebase and contribute more. Contributing by solving issues has really helped me understand the codebase better.
+- I wanted to work on the project `RFC 72: Background workers`, but now that it has been cancelled, I'm planning to work on `Greener coding: Wagtail’s climate impact`, the skillset required for it macthes mine, and the idea really interests me.
```
Comment (thibaudcolas) [inline 2023/Abdul Samad Ansari (Sam)/README.md:3]: Yay!

File: 2023/Abdul Samad Ansari (Sam)/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- [Django documentation](https://docs.djangoproject.com/en/4.1/) - You can not forget the documentation obviously.
+- [Corey Schafer Django Playlist](https://www.youtube.com/playlist?list=PL-osiE80TeTtoQCKZ03TU5fNfx2UY6U4p) - Another great resource for Django learners.
+- [Wagtail Documentation](https://docs.wagtail.org/en/stable/) - There is no place better than the official documentation to learn about Wagtail.
+- [Green coding](https://geekflare.com/green-coding/) - To learn about green coding
+- [Chris Adams Greening Django](https://www.youtube.com/watch?v=upiK4du5vUI0) - A video I found by doing some digging about Green coding.
+- [Green Metric Tool](https://github.com/green-coding-berlin/green-metrics-tool) - A developer tool is indented for measuring the energy consumption of software and doing life-cycle-analysis.
+- [Quantifying "greenness" of FaaS](https://www.linkedin.com/pulse/quantifying-greenness-faas-lukasz-mastalerz/) - From I learned how FasS can be useful in making coding greener.
```
Comment (thibaudcolas) [inline 2023/Abdul Samad Ansari (Sam)/README.md:58]: Excellent :) Chris’ talk is worth the watch.

File: 2023/Abdul Samad Ansari (Sam)/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+### Checklist: Django and Wagtail
+- [x] Go through Django’s [Getting started documentation and tutorial](https://docs.djangoproject.com/en/4.1/intro/), until part 7.
+- [x] Go through Wagtail’s Getting started tutorial: [Your first Wagtail site](https://docs.wagtail.org/en/stable/getting_started/tutorial.html)
+
+## **Research and learning resources**
+- [CS50W - Django](https://www.youtube.com/watch?v=w8q0C-C1js4&t=4925s) - This is a great resource to getting started with Django. Also do check out the successive video in the playlist where they teach about Models.
+- [Django documentation](https://docs.djangoproject.com/en/4.1/) - You can not forget the documentation obviously.
+- [Corey Schafer Django Playlist](https://www.youtube.com/playlist?list=PL-osiE80TeTtoQCKZ03TU5fNfx2UY6U4p) - Another great resource for Django learners.
```
Comment (thibaudcolas) [inline 2023/Abdul Samad Ansari (Sam)/README.md:53]: First time I see this one, interesting.

**Review by thibaudcolas — APPROVED** [review]
Nice one Sam!

File: 2023/Abdul Samad Ansari (Sam)/README.md (hunk 3 above)
Comment (SAMAD101) [reply 2023/Abdul Samad Ansari (Sam)/README.md:53]: Thank you for the feedback, sir.

**Pull request merged**